src/main/pg/motor.h

```diff
@@ -25,6 +25,14 @@
 #include "drivers/io.h"
 #include "drivers/dshot_bitbang.h"
 
+#if !defined(BRUSHED_MOTORS_PWM_RATE)
```

Comment:
Probably in .c only?

Reply:
Required for ValidateAndFixConfig in config/